Bluetooth/doc/BluetoothPlugin.md

```diff
@@ -956,13 +982,17 @@ When `true`, this device can be discovered by other Bluetooth devices.  When `fa
 ## *setName <sup>method</sup>*
 
 Sets the name of this device as seen by other Bluetooth devices.
+  
+### Events 
+
+  No Events.
 
 ### Parameters
 
 | Name | Type | Description |
 | :-------- | :-------- | :-------- |
 | params | object |  |
-| params.name | string | The name of the device |
+| params.name | string | Device name as specified by the manufacturer |
 
```

Comment:
"manufacturer" can be misleading here.
 it is a custom name, which can be the name given to the RDKTV or Xi or Xg device.
On Xg/Xi devices where we connect to XRE using ServiceManager, XRE-UI passed the DeviceName set by User for his/her Xg/Xi device to this API,
This is set as the name of the Bluetooth adaptor on every boot-cycle so that external devices like Smartphones/Tablets when scan for Bluetooth devices, see the name of Xg/Xi device

Reply:
Updated as suggested. ("Name of the Bluetooth Device")